Node Status Monitor/Python/api_helper.py

```diff
@@ -0,0 +1,108 @@
+import requests
```

Comment:
Check python 2.7 comments and reflect in Python folder.

Reply:
added